examples/siro_sandbox/episode_helper.py

```diff
@@ -2,9 +2,18 @@ class EpisodeHelper:
     def __init__(self, habitat_env):
         self._habitat_env = habitat_env
 
-        self._num_iter_episodes = float("inf") if habitat_env.episode_iterator.cycle else len(self._habitat_env.episode_iterator.episodes)  # type: ignore
+        self._episode_iterator_cycle = habitat_env.episode_iterator.cycle
+        self._num_iter_episodes: int = len(habitat_env.episode_iterator.episodes)  # type: ignore
         self._num_episodes_done: int = 0
 
+    @property
```

Comment:
I prefer the old code. The new code looks like needless complexity. The old code has the same functionality. The new code has the confusing disparity `self._num_iter_episodes != self.num_iter_episodes` in the cycle case, which may eventually cause a bug when someone misunderstands.

Reply:
Agree, reverted